cmd/kpod/push.go

```diff
@@ -29,6 +31,14 @@ var (
 			Name:  "cert-dir",
 			Usage: "`pathname` of a directory containing TLS certificates and keys",
 		},
+		cli.BoolFlag{
+			Name:  "compress",
+			Usage: "Compress tarball image layers when saving to directory using the 'dir' transport. (default is same compression type as source)",
```

Comment:
I think we decided to lowercase the first word in a Usage statement.  Regardless it should match the others in this file.  Suggest changing to:  "compress tarball image layers when saving to a directory using the 'dir' transport.  (default is the same compression type as the source)"

Reply:
fixed